### PR TITLE
Sam week2

### PR DESCRIPTION
**Pull request by Samtabanja**
still working on it !

### CODE REVIEW

File: week2/homework/src/data.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "todo": {
+        "1": "shopping",
```
Comment (J05J4) [inline week2/homework/src/data.json]: When you are making an object, it makes more sense to use something like
```
{
    "id": 1,
    "text": "shopping"
}
```
That way you can use `todo.id` and `todo.text`. If you use the id as key, it is actually similar to an array.

File: week2/homework/src/index1.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    case 'remove':
+        remove(todo)
+        break;
+    case 'list':
+        todolist()
+        break;
+    case 'reset':
+        reset()
+        break;
+    case 'update':
```
Comment (J05J4) [inline week2/homework/src/index1.js]: If I remove the third item with `node . remove 3` and then do `node . update 3 "asdasd"` a new item is created (with index 3), so the index is not updated if I remove an item. 
Maybe an easy solution could be to remove the id (number) altogether and just use an array, in that way the index updating is done automatically. In your json file you would have something like
```
todos: [
    { 
        "text": "shoping", 
        "done": false
    },
    {
        "text": "lunch",
        "done": false
    }
]
```
If you then want the first, you can do todos[0] and if you delete that one the second becomes the first automatically.

File: week2/homework/src/index.js [hunk 3]
```diff
@@ -1,3 +1,97 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const fs = require('fs')
```
Comment (J05J4) [inline week2/homework/src/index.js]: add semi colon `;`

File: week2/homework/src/index.js [hunk 4]
```diff
@@ -1,3 +1,97 @@
...
+    case 'reset':
+        reset()
+        break;
+    case 'update':
+        update(todo, todo2)
+        break;
+    case 'help':
+        help()
+        break;
+
```
Comment (J05J4) [inline week2/homework/src/index.js]: remove newlines

File: week2/homework/src/index1.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
+'use strict';
```
Comment (J05J4) [inline week2/homework/src/index1.js]: index.js is set as `"main"` in package.json, so this file is not used then?

File: week2/homework/src/data.json (hunk 1 above)
Comment (Samtabanja) [reply week2/homework/src/data.json]: thanks for this helpful feedbacks , all is done !

**Pull request closed without merging**